### ISSUE TITLE
about the exclusion loss

### ISSUE THREAD

**Issue by Liupengshuaige** [MarcoForte/FBA_Matting#13]
Hi, I have read your paper about the exclusion loss, does the exclusion loss can be written like this in pytorch:

(grad() is a function getting the H grad or V grad)
foreground_grad = torch.abs(grad(foreground)) 
background_grad = torch.abs(grad(background))
elemental_mul = foreground_grad.mul(background_grad)
loss = sum(elemental_mul)

Looking forward to your reply.

**Comment by MarcoForte** [comment]
Hi sorry for the delay, that looks good to me.

**Comment by kartikwar** [comment]
hey @Liupengshuaige can you please share the code to compute grad()
